lib/rake/package.rake

```diff
@@ -8,10 +8,6 @@ namespace :package do
     apps: {
       target: apps_dir('build/package'),
       symlink_name: 'blockly'
-    },
-    code_studio: {
-      target: code_studio_dir('build'),
-      symlink_name: 'code-studio'
```

Comment:
Here, we stop uploading or downloading the code-studio package, but does that actually clean any old code-studio package a frontend my have?  Might be worth making sure it's cleared out here.